webapp/views/App/views/Users/UserInvite/store/actions/useOnInvite.js

```diff
@@ -0,0 +1,49 @@
+import axios from 'axios'
+import { useDispatch } from 'react-redux'
+import { useHistory } from 'react-router'
+import * as R from 'ramda'
+
+import * as Validation from '@core/validation/validation'
+import * as UserInvite from '@core/user/userInvite'
+import { appModuleUri, userModules } from '@webapp/app/appModules'
+
+import { useSurveyCycleKey, useSurveyId } from '@webapp/store/survey'
+import { LoaderActions, NotificationActions } from '@webapp/store/ui'
+
+import { validateUserInvite } from './validate'
+
+export const useOnInvite = ({ userInvite }) => {
+  const dispatch = useDispatch()
+  const history = useHistory()
+  const surveyId = useSurveyId()
+  const surveyCycleKey = useSurveyCycleKey()
+
+  return () => {
+    ;(async () => {
```

Comment:
Every now and then I see this syntax in the code.
Is the semicolon required?
Could that be avoided with bracket-less arrow function?